modules/apps/headless/headless-builder/headless-builder-test/src/testFunctional/tests/APIBuilder-Multitenant/SupportMultitenancyForHeadlessBuilderRESTAPI.testcase

```diff
@@ -0,0 +1,88 @@
+@component-name = "portal-headless"
+definition {
+
+	property custom.properties = "feature.flag.LPS-167253=true${line.separator}feature.flag.LPS-178642=true";
+	property portal.release = "true";
+	property portal.upstream = "true";
+	property testray.component.names = "Object";
+	property testray.main.component.name = "Headless Builder";
+
+	setUp {
+		TestCase.setUpPortalInstance();
+
+		User.firstLoginUI();
+
+		task ("Given an published API application with endpoint and schema created in default instance") {
```

Comment:
dont we need these properties to run in default instance? 🤔 
```
property test.liferay.virtual.instance = "false";
		property test.run.type = "single";
```

Reply:
for these properties, if we use a virtual instance we don't need them

Reply:
oh yes, and the step is telling `Given an published API application with endpoint and schema created in default instance`, default instance, right?

Reply:
right, but in the next we create a virtual instance and log in to it, and all tests use the virtual instance

Reply:
The thing Meg point is that, we are using **in default instance**, so we actually need to create it in a default instance. And the way of `test.liferay.virtual.instance` is that:
- `test.liferay.virtual.instance` is set to true by default, this way tests will be run on the {server hostname} instance, not the default instance(localhost)
- when we set `test.liferay.virtual.instance = "false"`, tests will be run on the default instance
@Hijky hope this is helpful

Reply:
got it, thanks, I forgot these cases don't create anything in the virtual instance, will modify it on Monday

Reply:
yes, thanks @yanan for clarifying my thought, and sorry for not being clear from the beginning 😅 This was exactly the change I expected, having in mind that the step expresses creating stuff on the default instance :) Thanks for modifying it!